### ISSUE TITLE
from the twitterverse: "JASP contains a trojan" warning

### ISSUE THREAD

**Issue by EJWagenmakers** [jasp-stats/jasp-desktop#1537]
"Antivirus saying that JASP download contains a trojan."
For the latest version, on Windows. It would be good if Windows knew we are a trusted source. 
E.J.

**Comment by EJWagenmakers** [comment]
For the record, it was actually from the Forum.

**Comment by GernHuijberts** [comment]
F-secure safe reports malware in JASP 0.8.0.1.
![schermafdruk 2016-12-27 19 48 32](https://cloud.githubusercontent.com/assets/24324106/21506095/d0e90d34-cc6d-11e6-9b13-b8212252a5fb.png)

**Comment by FransMeerhoff** [comment]
Hello @GernHuijberts 

On your screenshot I can see that this see that this report comes from a MacOS
Unfortunately we are not able to reproduce this on a Mac Sierra with F-Secure

Can you help us with the following?

- We have run a scan on a Mac with OS Sierra and JASP 0.8.0.1 installed in the default folder and used F-Secure FASE Versie 16.4 (15923). I do not know F-Secure well but I assume that Virus information depends on the Dabaseversie. Can you tell us which versions of both you use?
- In the report screen I can not see the complete Filename, but it suggests that the message come from the Framework folder and that's where our R.Framework resides too. So we are interested in the complete filename. Can you open the report again and send us the complete names?
- Are you using special options running your scan, and does F-Secure give any more information about the found malware?

We would be very grateful if you could help us finding this pesky issue.

Kind Regards Frans Meerhoff
And best wishes for 2017

![fsecure](https://cloud.githubusercontent.com/assets/15656960/21602657/bceff4f4-d196-11e6-8e71-fd0fbcefc793.png)

**Comment by GernHuijberts** [comment]
Dear Frans,

Thank you for your reply.
I have just scanned the dmg file and F-secure does not give any warnings. I use the same F-secure version as you ( 16.4 (15923)) on OSX 10.11.6.
However, when instilling the program I got these warnings




Hope this helps to solve the issue.

thank you for your help

Gern Huijberts

**Comment by FransMeerhoff** [comment]
Dear @GernHuijberts 

I was able to reproduce the malware message from F-Secure.
After finding the more precise location, it was clear that the cause of the malware message came from the pbkrtest  R.package we ship. It is the pbkrtest-introduction.pdf file in the doc folder of this package that causes this issue. It is definitely innocent but I've reported this issue anyway to both the maintainer of this package (sorenh@math.aau.dk) and F-Secure and consider the issue as closed.

Thanks for your help
Kind regards Frans Meerhoff



<img width="1288" alt="screen shot 2017-01-05 at 16 50 59" src="https://cloud.githubusercontent.com/assets/15656960/21689619/2a6b59de-d371-11e6-9129-987f044ee752.png">
